src/cmd/create.go

```diff
@@ -63,6 +63,8 @@ var (
 		{"/etc/profile.d/toolbox.sh", "/etc/profile.d/toolbox.sh"},
 		{"/etc/profile.d/toolbox.sh", "/usr/share/profile.d/toolbox.sh"},
 	}
+
+	errorNotTerminal = errors.New("terminal is not connected")
```

Comment:
We can use [unix.ENOTTY](https://pkg.go.dev/golang.org/x/sys/unix) instead of inventing our own.  It's the same error that `term.GetState` returns when the `*os.File` isn't connected to a terminal device.

Or we can create the final user facing error directly inside `pullImage` because it's a sufficiently high-level function in the `cmd` package.